### PR TITLE
Update core_benchmarks.md

### PR DESCRIPTION
**Pull request by gareth-ellis**
Adding extra INFO for the "RUNS" parameter, now that is available through Jenkins

### CODE REVIEW

File: docs/core_benchmarks.md [hunk 1]
```diff
@@ -6,15 +6,21 @@ The current jenkins job aims at allowing the impact of a PR on a particular bran
...
 
 ### CATEGORY
 This is a category of benchmarks in the node/benchmark folder from core
+
 ### FILTER
 This reduces further the benchmarks in scope and is includes benchmark where FILTER exists in the filename.
 
-On running, a minimum of 30 iterations of each benchmark/variation per build, once they have all finished Rscript will summarise the results, suggesting improvement / regression as well as confidence intervals.
+### RUNS
+This changes the number of runs for per benchmark to run, the default is 30 if this is value is left empty.
```
Comment (vsemozhetbyt) [inline docs/core_benchmarks.md]: Nits: `for per` -> `per`? `if this is` -> `if this`?

File: docs/core_benchmarks.md [hunk 2]
```diff
@@ -6,15 +6,21 @@ The current jenkins job aims at allowing the impact of a PR on a particular bran
...
 This is a category of benchmarks in the node/benchmark folder from core
+
 ### FILTER
 This reduces further the benchmarks in scope and is includes benchmark where FILTER exists in the filename.
 
-On running, a minimum of 30 iterations of each benchmark/variation per build, once they have all finished Rscript will summarise the results, suggesting improvement / regression as well as confidence intervals.
+### RUNS
+This changes the number of runs for per benchmark to run, the default is 30 if this is value is left empty. 
+
+On running, a minimum of 30 iterations  of each benchmark/variation per build (unless RUNS is defined), once they have all finished Rscript will summarise the results, suggesting improvement / regression as well as confidence intervals.
```
Comment (vsemozhetbyt) [inline docs/core_benchmarks.md]: Nits:
does `, a minimum of 30 iterations of each benchmark/variation per build (unless RUNS is defined),` lack a verb?
`iterations  of` — a redundant space.

**Comment by gareth-ellis** [comment]
Nits addressed, PTAL

**Comment by mhdawson** [comment]
Landed as 6e5f7d2aac8b8144ec5ce677854ac36860e8561e

**Pull request closed without merging**